.github/workflows/rollback.yml

```diff
@@ -0,0 +1,82 @@
+name: Rollback release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-to-roll-back:
+        description: version number that needs to be rolled back (format x.x.x)
+        type: string
+        required: true
+
+jobs:
+  determine-target-tag:
```

Comment:
I think we need a check to determine if `github.ref_name` is a tag instead of a branch.

Reply:
Ah good idea. Thanks.